classes/rubygentest.bbclass

```diff
@@ -58,4 +58,4 @@ python() {
         d.setVarFlag("do_rubygems_gen_test", "nostamp", "1")
     else:
         d.setVarFlag("do_rubygems_gen_test", "noexec", "1")
-}
\ No newline at end of file
+}
```

Comment:
potential issue: Multiple fragments of the same function in the same file should be merged